runtime/lua/vim/lsp/util.lua

```diff
@@ -1391,6 +1392,7 @@ function M.open_floating_preview(contents, syntax, opts)
   opts.wrap = opts.wrap ~= false -- wrapping by default
   opts.stylize_markdown = opts.stylize_markdown ~= false
   opts.close_events = opts.close_events or {"CursorMoved", "CursorMovedI", "BufHidden", "InsertCharPre"}
+  opts.enter = opts.enter == true
```

Comment:
truthy values (such as a lua number) should either coerce to `true` or to an error but not `false`.

Reply:
I think that it would make more sense for this value to default to false as in most cases, it is required to be false and only in a few cases would someone want it to be true.

Is it a standard followed by the `neovim` codebase? If so, I am ready to change it!